o1vm/src/pickles/prover.rs

```diff
@@ -194,14 +306,14 @@ where
     let alpha: G::ScalarField = fq_sponge.challenge();
 
     let zk_rows = 0;
-    let column_env: ColumnEnvironment<'_, G::ScalarField> = {
+    let column_env: ColumnEnvironment<'_, G::ScalarField, ID> = {
         // FIXME: use a proper Challenge structure
         let challenges = BerkeleyChallenges {
             alpha,
-            // No permutation argument for the moment
+            // TODO: No permutation argument for the moment
             beta: G::ScalarField::zero(),
             gamma: G::ScalarField::zero(),
-            // No lookup for the moment
+            // TODO: No lookup for the moment
```

Comment:
I thought the join combiner was generated during the lookup env creation.
I'd say it should be added here